tests/functionaltests/FunctionalTests.cpp

```diff
@@ -691,7 +697,7 @@ class ProjectionTest {
     }
 
     TEST_METHOD_CLEANUP(ProjectionTestCleanup) {
-        FunctionalTestCleanupUIApplication();
+        (void)FrameworkHelper::RunOnUIThread(&FunctionalTestCleanupUIApplication);
```

Comment:
I assumed that the return result may have affected the outcome of the test. However, it doesn't so I will move to test the HRESULT as well.

---
In reply to: [94668846](https://github.com/Microsoft/WinObjC/pull/1627#discussion_r94668846) [](ancestors = 94668846,94668773,94668735)